lib/evss/gi_bill_status/service.rb

```diff
@@ -39,6 +51,12 @@ def get_gi_bill_status
         response = OpenStruct.new(status: e.status, body: e.body)
         EVSS::GiBillStatus::GiBillStatusResponse.new(response.status, response)
       end
+
+      def self.current_time
+        @time ||= Time.now.in_time_zone(OPERATING_ZONE)
```

Comment:
this variable will never be updated, is that intended?

Reply:
Yes, I want it memoized per user request since it is used by three other methods in this class.

Reply:
are you sure it gets reset after the request is finished?

Reply:
You're right!   Didn't take into account it was a class method, obviously.  Thanks.